property.ts

```diff
@@ -11,7 +11,16 @@ import {
   isTraitReferenceFunction,
 } from "./shared";
 import { dim, green, red, underline, yellow } from "ansicolor";
-import { ContractInterfaceFunction } from "@hirosystems/clarinet-sdk-wasm";
+import {
+  ContractInterfaceFunction,
+  ContractInterfaceFunctionAccess,
+  ContractInterfaceFunctionArg,
+  ContractInterfaceFunctionOutput,
+  List,
+  TraitReference,
+} from "@hirosystems/clarinet-sdk-wasm";
+import { writeFileSync } from "fs";
+import { EnrichedContractInterfaceFunction } from "./shared.types";
```

Comment:
I keep not being able to fully follow the purpose of `EnrichedContractInterfaceFunction`. Is it a union?